rest/models/markets.go

```diff
@@ -11,6 +12,13 @@ type GetMarketHolidaysParams struct{}
 // GetMarketHolidaysResponse is the response returned by the GetMarketHolidays method.
 type GetMarketHolidaysResponse []MarketHoliday
 
+// todo: this endpoint is unlikely to ever have params so should we delete this type?
+// GetMarketStatusParams is the set of parameters for the GetMarketStatus method.
+type GetMarketStatusParams struct{}
+
+// GetMarketStatusResponse is the response returned by the GetMarketStatus method.
+type GetMarketStatusResponse MarketStatus
```

Comment:
i think we should delete the `MarketStatus` type and move its fields into this struct instead (`MarketHoliday` was only done this way because the response is a slice of market holidays)